stable/aws-iam-authenticator/templates/daemonset.yaml

```diff
@@ -1,22 +1,30 @@
 ---
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: {{ template "aws-iam-authenticator.fullname" . }}
   labels:
-    k8s-app: {{ template "aws-iam-authenticator.name" . }}
+    app.kubernetes.io/name: {{ include "aws-iam-authenticator.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ include "aws-iam-authenticator.chart" . }}
   annotations:
     checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
 spec:
   updateStrategy:
     type: RollingUpdate
+  selector:
+    matchLabels:
```

Comment:
Is this change necessary for the new API version? I'm not sure I've seen this before.

Reply:
It was failing e2e tests without this on my local machine. I saw where this is in the CONTRIBUTING doc as a standard so I changed it. Should I add back the old label `k8s-app`?

Reply:
It's fine with me.